### PR TITLE
Pa poc headline limit

### PR DESCRIPTION
**Pull request by eozan**

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

File: superdesk/metadata/item.py [hunk 1]
```diff
@@ -332,7 +332,8 @@
         'type': 'datetime',
         'versioned': False
     },
-    'lock_session': Resource.rel('auth')
+    'lock_session': Resource.rel('auth'),
+    'template_id': Resource.rel('content_templates', embeddable=False, nullable=True)
```
Comment (petrjasek) [inline superdesk/metadata/item.py]: can you call this `template` ? we already set that field for item created on schedule so it would work together

**Comment by plamut** [comment]
Looks good, just consider @petrjasek's recommendation if it applies.

:+1:

**Comment by petrjasek** [comment]
obsolete

**Pull request closed without merging**